lms/static/sass/discussion/_layouts.scss

```diff
@@ -0,0 +1,40 @@
+// Layouts for discussion pages
+
+section.user-profile {
+    background-color: $sidebar-color;
+
+    .user-profile {
+        padding: 32px 36px;
+        min-height: 500px;
+    }
+
+    .sidebar-username {
+        font-weight: 700;
```

Comment:
Not straightforwardly as this sass is used in the courseware as well where the UXPL variables aren't available. Note that I only reindented this file, and am otherwise leaving the user profile styled as is for this PR. We will have a future epic to clean up the user profile pages.